sale_commission/models/sale_order.py

```diff
@@ -84,7 +83,7 @@ def onchange_agent(self):
     @api.depends('sale_line.price_subtotal')
     def _compute_amount(self):
         for line in self:
-            line.amount = 0.0
+            amount = 0.0
```

Comment:
This is not needed, as these values are cached and you don't have performance loss in this case.